sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -139,9 +139,56 @@ Applications must make sure that the Level-Zero handles themselves aren't used s
 Practically speaking, and taking into account that SYCL runtime takes ownership of the Level-Zero handles,
 the application should not attempt further direct use of those handles.
 
+## 5 Level-Zero additional functionality
+
+### 5.1 Device Information Descriptors
+The Level Zero backend provides the following device information descriptors
+that an application can use to query information about a Level Zero device.
+Applications use these queries via the `device::get_backend_info<>()` member
+function as shown in the example below (which illustrates the `free_memory`
+query):
+
+``` C++
+sycl::queue Queue;
+auto Device = Queue.get_device();
+
+size_t freeMemory =
+  Device.get_backend_info<sycl::ext::oneapi::level_zero::info::device::free_memory>();
+```
+
+New descriptors added as part of this specification are found in the table below:
+
+| Descriptor | Description |
+| ---------- | ----------- |
+| `sycl::ext::oneapi::level_zero::info::device::free_memory` | Returns the number of bytes of free memory for the device. |
+
+
+#### 5.1.1 Free Memory Query Details

```

Comment:
And then delete this heading.

Now we can easily add new descriptors in the future by adding a row to the table and a declaration to the synopsis.